### PR TITLE
IBX-5948: Product page URLs

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-5948
| Versions      | master
| Edition       | Content

Product Page URLs added in PIM.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/pim/work_with_product_page_urls.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Color
+- Integer
+- Measurement
+- Selection
+
+You can use attributes identifiers in the URL alias pattern field. 
+A customized URL is short and easy to remember. 
+
+It can help to understand what the product is about just by reading the link. 
+It improves the user experience and reduces bounce rates on the website.
```
Comment (SylvainGuittard) [inline docs/pim/work_with_product_page_urls.md:26]: I think this part can be moved at the top of the page

File: docs/pim/work_with_product_page_urls.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+description: You can create manage the system URLs for product items. 
+---
+
+# Product page URLs
+
+Every [Product](products.md) has its own URLs that is automatically generated. It is based on a pattern that says which field or fields are used to build the URL (URL alias name pattern).
+Product can have multiple URLs if it is included in many catalogs and exposed to many sites.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/pim/work_with_product_page_urls.md:8]: As discussed with @kisztof - sentence deleted.

File: docs/pim/work_with_product_page_urls.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+![URL attributes](url_attributes.png)
+
+Now, you can see new URL alias name pattern in the Product Type's view.
+
+![URL alias name pattern.png](url_alias_name_pattern.png)
+
+## Product Attributes identifiers
+
+Products have their own [attributes](products.md#attributes) that provide different information about them.
```
Comment (SylvainGuittard) [inline docs/pim/work_with_product_page_urls.md:21]:
```suggestion
Products have their own [attributes](products.md#attributes) to define product specifications. 
```

File: docs/pim/work_with_product_page_urls.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+The following attribute types can be used in URL alias name pattern field:
+
+- Checkbox
+- Color
+- Integer
+- Selection
+- Measurement (single)
+- Measurement (range)
+
+!!! note
```
Comment (SylvainGuittard) [inline docs/pim/work_with_product_page_urls.md:32]: Not sure we want to keep this :)

File: docs/pim/work_with_product_page_urls.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    Float atribute is not available while creating URL alias name pattern.
+
+| Type | Returned value | Example|
+|---|---|---|
+| Checkbox | Label of the checkbox is checked | wireless |
+| Color | Hex code value | 8b0000 |
+| Integer | Integer | 26 |
+| Measurement (single) | Single value and unit | 256gb |
+| Measurement (range) | Value unit and minimum/maximum values | 20-25cm|
+| Selection | Selected value with configured separator | children-infants |
```
Comment (SylvainGuittard) [inline docs/pim/work_with_product_page_urls.md:43]:
```suggestion
| Type | Returned value | Example|
|---|---|---|
| Checkbox | Label of the checkbox is checked | wireless |
| Color | Hex code value | 8b0000 |
| Integer | Integer | 26 |
| Selection | Selected value with configured separator | children-infants |
| Measurement (single) | Single value and unit | 256gb |
| Measurement (range) | Value unit and minimum/maximum values | 20-25cm|
```
Changed the order so it's consistent with the list above

File: docs/pim/work_with_product_page_urls.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+You can use many attributes identifiers as well as fields identifiers in the URL alias pattern field. 
+
+Having keyword-rich URLs also improves the product's visibility in search engine results and boost the page's ranking in search results.
```
Comment (SylvainGuittard) [inline docs/pim/work_with_product_page_urls.md:47]:
```suggestion
Having keyword-rich URLs also improves the product's visibility in search engine results and boosts the page's ranking in search results.
```

**Review by SylvainGuittard — COMMENTED** [review]
Few changes to make it perfect :)

File: docs/pim/work_with_product_page_urls.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
+---
+description: You can create URL alias name pattern and custom URLs for the Products.
+---
+
+# Product page URLs
+
+Every [Product item](products.md) has a system URL that is automatically generated. It is based on a pattern that combines fields and attributes to build the URL (URL alias name pattern).
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:7]:
```suggestion
Every [product item](products.md) has a system URL that is automatically generated. 
It is based on a pattern that combines fields and attributes to build the URL (URL alias name pattern).
```

File: docs/pim/work_with_product_page_urls.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+When you are creating new [Product Type](create_product_types.md) you can set up Product URL alias name pattern.
+
+To do it, fill in all necessary information and choose the attributes that you want to use in URL alias name pattern field and click **Save** button.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:11]:
```suggestion
To do it, fill in all the necessary information and choose the attributes that you want to use in URL alias name pattern field and click **Save** button.
```

**Review by SylvainGuittard — APPROVED** [review]

File: docs/pim/work_with_product_page_urls.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+| Measurement (range) | Value unit and minimum/maximum values | 20-25cm|
+
+You can use many attributes identifiers as well as fields identifiers in the URL alias pattern field. 
+
+Having keyword-rich URLs also improves the product's visibility in search engine results and boosts the page's ranking in search results.
+
+## Create custom URL
+
+You can create custom URL for each Product.
+Customized URLs are easy to remember, help for SEO optimatization and reduce bounce rates on the website.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:48]:
```suggestion
Customized URLs are easy to remember, help with SEO optimization and reduce bounce rates on the website.
```

File: docs/pim/work_with_product_page_urls.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+4\. Click **Add new** button.
+
+5\. Provide custom URL in "URL" field and set up all necessary settings.
+
+![Create custom URL](create_custom_url.png)
+
+6\. Click **Create**.
+
+Now, in the **URL** tab in the product's view, you can see new custom URL for the Product.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:69]:
```suggestion
Now, in the **URL** tab in the product's view, you can see new custom URL for the product.
```

File: docs/pim/work_with_product_page_urls.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+You can create custom URL for each Product.
+Customized URLs are easy to remember, help for SEO optimatization and reduce bounce rates on the website.
+It improves the user experience - you can understand what the product is about just by reading the link. 
+
+To do it, follow the steps:
+
+1\. In the left panel, go to **Product catalog** -> **Products**.
+
+2\. Choose the Product from the list.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:55]:
```suggestion
2\. Choose a product from the list.
```

File: docs/pim/work_with_product_page_urls.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+| Measurement (single) | Single value and unit | 256gb |
+| Measurement (range) | Value unit and minimum/maximum values | 20-25cm|
+
+You can use many attributes identifiers as well as fields identifiers in the URL alias pattern field. 
+
+Having keyword-rich URLs also improves the product's visibility in search engine results and boosts the page's ranking in search results.
+
+## Create custom URL
+
+You can create custom URL for each Product.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:47]:
```suggestion
You can create custom URL for each product.
```

File: docs/pim/work_with_product_page_urls.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
+---
+description: You can create URL alias name pattern and custom URLs for the Products.
+---
+
+# Product page URLs
+
+Every [Product item](products.md) has a system URL that is automatically generated. It is based on a pattern that combines fields and attributes to build the URL (URL alias name pattern).
+
+When you are creating new [Product Type](create_product_types.md) you can set up Product URL alias name pattern.
```
Comment (MagdalenaZuba) [inline docs/pim/work_with_product_page_urls.md:9]:
```suggestion
When you are creating new [Product Type](create_product_types.md) you can set up product URL alias name pattern.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**